packages/plugins/foreman-discovery-image-service/foreman-discovery-image-service.spec

```diff
@@ -1,15 +1,18 @@
 Name: foreman-discovery-image-service
 Version: 1.0.0
-Release: 3%{?dist}
+Release: 4%{?dist}
 Summary: Metapackage with dependencies for FDI
 
 Group: Applications/System
 License: GPLv2+
 URL: https://github.com/theforeman/foreman-discovery-image
 
+# explicitly define, as we build on top of an scl, not inside with scl_package 
+%{?scl:%global scl_prefix %{scl}-} 
+
 Requires:	foreman-proxy
 %if 0%{?rhel} == 7
-Requires:	tfm-rubygem-smart_proxy_discovery_image
+Requires:	%{?scl_prefix}rubygem-smart_proxy_discovery_image
 %else
 Requires:	rubygem-smart_proxy_discovery_image
 %endif
```

Comment:
This can be folded into the single `Requires` line. No need for the RHEL 7 check anymore since `%{?scl_prefix}` is just empty on EL8.